README.md

```diff
@@ -52,6 +52,9 @@ $ curl -H "Content-type: application/json" -X POST \
 
 ## Configuration
 
+Attention: We have introduced a new behavior in [#79](https://github.com/prometheus-community/jiralert/pull/79) for building the string that is used as an issue label in jira so jiralert can find an issue it opened. The new behavior ensures that at all times the string is a legal issue label in the jira api. **This change is opt-in** to avoid breaking users upgrading from older releases, but we will remove the old behavior + flag in a future release.
```

Comment:
That should probably go to CHANGELOG.md instead 🤗

Reply:
- #83 adds a changelog and is now a prerquisite for this pr
- this pr adds a new entry in the changelog